2020-05-29 16:05:47.603736: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:187: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\research\neural_stack.py:52: The name tf.nn.rnn_cell.RNNCell is deprecated. Please use tf.compat.v1.nn.rnn_cell.RNNCell instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

INFO:tensorflow:Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
I0529 16:05:58.463809 17704 usr_dir.py:43] Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
INFO:tensorflow:Loading hparams from existing json ./output\hparams.json
I0529 16:05:58.466800 17704 hparams_lib.py:64] Loading hparams from existing json ./output\hparams.json
INFO:tensorflow:Overwrite key batch_size: 4096 -> 256
I0529 16:05:58.475776 17704 hparams_lib.py:86] Overwrite key batch_size: 4096 -> 256
INFO:tensorflow:Overriding hparams in transformer_small with batch_size=128
I0529 16:05:58.476775 17704 hparams_lib.py:55] Overriding hparams in transformer_small with batch_size=128
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

W0529 16:05:58.476775 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

W0529 16:05:58.486778 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

W0529 16:05:58.486778 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0529 16:05:58.486778 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
W0529 16:05:58.486778 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

W0529 16:05:58.486778 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

INFO:tensorflow:Configuring DataParallelism to replicate the model.
I0529 16:05:58.487745 17704 trainer_lib.py:264] Configuring DataParallelism to replicate the model.
INFO:tensorflow:schedule=train
I0529 16:05:58.487745 17704 devices.py:76] schedule=train
INFO:tensorflow:worker_gpu=1
I0529 16:05:58.487745 17704 devices.py:77] worker_gpu=1
INFO:tensorflow:sync=False
I0529 16:05:58.487745 17704 devices.py:78] sync=False
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

W0529 16:05:58.487745 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

WARNING:tensorflow:Schedule=train. Assuming that training is running on a single machine.
W0529 16:05:58.487745 17704 devices.py:141] Schedule=train. Assuming that training is running on a single machine.
INFO:tensorflow:datashard_devices: ['gpu:0']
I0529 16:05:58.487745 17704 devices.py:170] datashard_devices: ['gpu:0']
INFO:tensorflow:caching_devices: None
I0529 16:05:58.487745 17704 devices.py:171] caching_devices: None
INFO:tensorflow:ps_devices: ['gpu:0']
I0529 16:05:58.487745 17704 devices.py:172] ps_devices: ['gpu:0']
I0529 16:05:58.488747 17704 merge_vocab.py:101] Loading and processing vocabulary from: C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP\usr_dir/merge.txt.vocab.clean
I0529 16:05:58.495761 17704 merge_vocab.py:105] Done
INFO:tensorflow:Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x00000254559CFE48>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x0000025455ADED08>}
I0529 16:05:58.496758 17704 estimator.py:212] Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x00000254559CFE48>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x0000025455ADED08>}
WARNING:tensorflow:Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x0000025455AC7D38>) includes params argument, but params are not passed to Estimator.
W0529 16:05:58.497719 17704 model_fn.py:630] Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x0000025455AC7D38>) includes params argument, but params are not passed to Estimator.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

W0529 16:05:58.497719 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

W0529 16:05:58.498754 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
W0529 16:05:58.549669 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
INFO:tensorflow:Reading data files from ./data_dir\translate_up2down-train*
I0529 16:05:58.564594 17704 problem.py:651] Reading data files from ./data_dir\translate_up2down-train*
INFO:tensorflow:partition: 0 num_data_files: 9
I0529 16:05:58.568583 17704 problem.py:677] partition: 0 num_data_files: 9
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
W0529 16:05:58.576560 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

W0529 16:05:58.742629 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

W0529 16:05:59.113636 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
W0529 16:06:00.187210 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:06:00.400855 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
W0529 16:06:00.755878 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

W0529 16:06:01.515002 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:06:02.014678 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
INFO:tensorflow:Calling model_fn.
I0529 16:06:02.367400 17704 estimator.py:1148] Calling model_fn.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0529 16:06:02.368390 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

W0529 16:06:02.368390 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

W0529 16:06:02.368390 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

W0529 16:06:02.382362 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

INFO:tensorflow:Setting T2TModel mode to 'train'
I0529 16:06:02.383358 17704 t2t_model.py:2262] Setting T2TModel mode to 'train'
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0529 16:06:02.383358 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

W0529 16:06:02.497536 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

INFO:tensorflow:Using variable initializer: uniform_unit_scaling
I0529 16:06:03.472649 17704 api.py:332] Using variable initializer: uniform_unit_scaling
INFO:tensorflow:Transforming feature 'inputs' with symbol_modality_3486_256.bottom
I0529 16:06:13.556497 17704 api.py:332] Transforming feature 'inputs' with symbol_modality_3486_256.bottom
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
W0529 16:06:14.994766 17704 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0529 16:06:15.504242 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

INFO:tensorflow:Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
I0529 16:06:17.904666 17704 api.py:332] Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
INFO:tensorflow:Building model body
I0529 16:06:18.240387 17704 api.py:332] Building model body
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

W0529 16:06:23.266855 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0529 16:06:23.570676 17704 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:Entity <function attention_bias_to_padding at 0x000002544ECF2CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
W0529 16:06:25.015279 17704 ag_logging.py:146] Entity <function attention_bias_to_padding at 0x000002544ECF2CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

W0529 16:06:27.078061 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

W0529 16:06:27.079020 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

W0529 16:06:33.308841 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

W0529 16:06:33.309836 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:Entity <function layers at 0x0000025445687438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x00000254574046C8>
W0529 16:06:33.663921 17704 ag_logging.py:146] Entity <function layers at 0x0000025445687438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x00000254574046C8>
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

W0529 16:06:33.664862 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

W0529 16:06:34.858857 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

INFO:tensorflow:Transforming body output with symbol_modality_3486_256.top
I0529 16:06:43.214311 17704 api.py:332] Transforming body output with symbol_modality_3486_256.top
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

W0529 16:06:44.978103 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

W0529 16:06:50.959979 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

INFO:tensorflow:Base learning rate: 2.000000
I0529 16:06:53.746947 17704 learning_rate.py:29] Base learning rate: 2.000000
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

W0529 16:06:53.755885 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

INFO:tensorflow:Trainable Variables Total size: 4581888
I0529 16:06:53.760906 17704 optimize.py:336] Trainable Variables Total size: 4581888
INFO:tensorflow:Non-trainable variables Total size: 5
I0529 16:06:53.760906 17704 optimize.py:336] Non-trainable variables Total size: 5
INFO:tensorflow:Using optimizer adam
I0529 16:06:53.761868 17704 optimize.py:191] Using optimizer adam
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

W0529 16:06:53.762904 17704 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

INFO:tensorflow:Done calling model_fn.
I0529 16:06:58.217503 17704 estimator.py:1150] Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
I0529 16:06:58.218500 17704 basic_session_run_hooks.py:541] Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
I0529 16:07:10.318166 17704 monitored_session.py:240] Graph was finalized.
2020-05-29 16:07:10.328257: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-05-29 16:07:10.340562: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-05-29 16:07:11.081280: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce MX150 major: 6 minor: 1 memoryClockRate(GHz): 1.0375
pciBusID: 0000:01:00.0
2020-05-29 16:07:11.081758: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-05-29 16:07:11.117305: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:07:11.152108: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-05-29 16:07:11.167137: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-05-29 16:07:11.216513: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-05-29 16:07:11.247598: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-05-29 16:07:11.338794: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-29 16:07:11.615764: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-05-29 16:07:14.625635: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-05-29 16:07:14.626022: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-05-29 16:07:14.626271: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-05-29 16:07:14.631188: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1945 MB memory) -> physical GPU (device: 0, name: GeForce MX150, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from ./output\model.ckpt-0
I0529 16:07:14.655258 17704 saver.py:1284] Restoring parameters from ./output\model.ckpt-0
2020-05-29 16:07:15.352511: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.90G (2040109312 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-05-29 16:07:15.353302: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.71G (1836098304 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
W0529 16:07:15.649842 17704 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
I0529 16:07:16.103629 17704 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0529 16:07:16.301102 17704 session_manager.py:502] Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into ./output\model.ckpt.
I0529 16:07:35.974130 17704 basic_session_run_hooks.py:606] Saving checkpoints for 0 into ./output\model.ckpt.
2020-05-29 16:07:48.700449: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:07:49.485736: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.487051: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.487342: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.487875: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.488329: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.488839: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.495667: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.496125: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.500095: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.500414: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.504061: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.506728: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.507121: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.507490: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.508938: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.510649: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.511458: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.513162: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.513479: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.513789: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.514100: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.515468: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.516885: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.517551: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.519092: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.520225: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.521617: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.523270: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.525013: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.526267: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.527685: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.528423: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.529712: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.532324: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.532826: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.538569: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.539135: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.539667: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.540036: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.541548: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.541873: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.542470: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.542848: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.543431: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.543757: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.544200: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.544682: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.545804: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.546236: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.546740: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.547163: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.547683: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.548011: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.548368: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.548679: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.549791: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.550163: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.550592: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.550912: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.552130: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.552489: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.553663: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.554938: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.555890: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.556526: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.557031: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.557395: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.559554: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.559943: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.560413: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.560734: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.561368: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.561792: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.562183: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.562494: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.564193: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.564603: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.565039: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.565403: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.566804: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.567173: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.567779: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.568093: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.568862: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.569301: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.571305: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.571667: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.572792: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.573215: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.573637: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.574057: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.575311: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.575628: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.576082: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.576390: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.577036: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.577345: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.577833: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:49.578148: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:50.633708: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:50.634519: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:07:50.634927: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
2020-05-29 16:07:50.634702: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
Traceback (most recent call last):
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1365, in _do_call
    return fn(*args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(104, 256), b.shape=(256, 256), m=104, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(104, 256), b.shape=(256, 256), m=104, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1195, in _train_model_default
    saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1494, in _train_with_estimator_spec
    _, loss = mon_sess.run([estimator_spec.train_op, estimator_spec.loss])
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 754, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1259, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1360, in run
    raise six.reraise(*original_exc_info)
  File "D:\Anaconda\envs\NLP\lib\site-packages\six.py", line 703, in reraise
    raise value
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1345, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1418, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1176, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 956, in run
    run_metadata_ptr)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1180, in _run
    feed_dict_tensor, options, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1359, in _do_run
    run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1384, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(104, 256), b.shape=(256, 256), m=104, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(104, 256), b.shape=(256, 256), m=104, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
0 successful operations.
0 derived errors ignored.

Original stack trace for 'transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul':
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1191, in _train_model_default
    features, labels, ModeKeys.TRAIN, self.config)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1149, in _call_model_fn
    model_fn_results = self._model_fn(features=features, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1420, in wrapping_model_fn
    use_tpu=use_tpu)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1485, in estimator_model_fn
    logits, losses_dict = model(features)  # pylint: disable=not-callable
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 325, in call
    sharded_logits, losses = self.model_fn_sharded(sharded_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 365, in model_fn_sharded
    if self.use_body_sharded():
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 402, in model_fn_sharded
    sharded_logits, sharded_losses = dp(self.model_fn, datashard_to_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 171, in __call__
    for i in range(self.n):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 229, in __call__
    if self._devices[i] != DEFAULT_DEV_STRING:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 231, in __call__
    outputs.append(fns[i](*my_args[i], **my_kwargs[i]))
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 429, in model_fn
    body_out = self.body(transformed_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 243, in body
    if self.has_input:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 246, in body
    encoder_output, encoder_decoder_attention_bias = self.encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 200, in encode
    return transformer_encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 103, in transformer_encode
    encoder_output = encoder_function(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 201, in transformer_encoder
    for layer in range(hparams.num_encoder_layers or hparams.num_hidden_layers):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 212, in transformer_encoder
    y = common_attention.multihead_attention(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4649, in multihead_attention
    if cache is None or memory_antecedent is None:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4650, in multihead_attention
    q, k, v = compute_qkv(query_antecedent, memory_antecedent,
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4454, in compute_qkv
    q = compute_attention_component(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4399, in compute_attention_component
    if vars_3d_num_heads > 0:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4414, in compute_attention_component
    if filter_width == 1:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4415, in compute_attention_component
    return common_layers.dense(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py", line 3083, in dense
    activations = layers().Dense(units, **kwargs)(x)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\layers\core.py", line 1039, in call
    outputs = standard_ops.tensordot(inputs, self.kernel, [[rank - 1], [0]])
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 4071, in tensordot
    ab_matmul = matmul(a_reshape, b_reshape)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\dispatch.py", line 180, in wrapper
    return target(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 2754, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\gen_math_ops.py", line 6136, in mat_mul
    name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()

2020-05-29 16:10:27.918191: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:187: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\research\neural_stack.py:52: The name tf.nn.rnn_cell.RNNCell is deprecated. Please use tf.compat.v1.nn.rnn_cell.RNNCell instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

INFO:tensorflow:Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
I0529 16:10:37.549966 12152 usr_dir.py:43] Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
INFO:tensorflow:Loading hparams from existing json ./output\hparams.json
I0529 16:10:37.552958 12152 hparams_lib.py:64] Loading hparams from existing json ./output\hparams.json
INFO:tensorflow:Overwrite key batch_size: 4096 -> 128
I0529 16:10:37.568921 12152 hparams_lib.py:86] Overwrite key batch_size: 4096 -> 128
INFO:tensorflow:Overriding hparams in transformer_small with batch_size=32
I0529 16:10:37.569918 12152 hparams_lib.py:55] Overriding hparams in transformer_small with batch_size=32
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

W0529 16:10:37.569918 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

W0529 16:10:37.584845 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

W0529 16:10:37.584845 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0529 16:10:37.584845 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
W0529 16:10:37.584845 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

W0529 16:10:37.585842 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

INFO:tensorflow:Configuring DataParallelism to replicate the model.
I0529 16:10:37.585842 12152 trainer_lib.py:264] Configuring DataParallelism to replicate the model.
INFO:tensorflow:schedule=train
I0529 16:10:37.585842 12152 devices.py:76] schedule=train
INFO:tensorflow:worker_gpu=1
I0529 16:10:37.585842 12152 devices.py:77] worker_gpu=1
INFO:tensorflow:sync=False
I0529 16:10:37.585842 12152 devices.py:78] sync=False
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

W0529 16:10:37.585842 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

WARNING:tensorflow:Schedule=train. Assuming that training is running on a single machine.
W0529 16:10:37.585842 12152 devices.py:141] Schedule=train. Assuming that training is running on a single machine.
INFO:tensorflow:datashard_devices: ['gpu:0']
I0529 16:10:37.585842 12152 devices.py:170] datashard_devices: ['gpu:0']
INFO:tensorflow:caching_devices: None
I0529 16:10:37.585842 12152 devices.py:171] caching_devices: None
INFO:tensorflow:ps_devices: ['gpu:0']
I0529 16:10:37.585842 12152 devices.py:172] ps_devices: ['gpu:0']
I0529 16:10:37.586839 12152 merge_vocab.py:101] Loading and processing vocabulary from: C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP\usr_dir/merge.txt.vocab.clean
I0529 16:10:37.596847 12152 merge_vocab.py:105] Done
INFO:tensorflow:Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x00000253E3DA6B48>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x00000253E3E486C8>}
I0529 16:10:37.599805 12152 estimator.py:212] Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x00000253E3DA6B48>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x00000253E3E486C8>}
WARNING:tensorflow:Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x00000253E42B7D38>) includes params argument, but params are not passed to Estimator.
W0529 16:10:37.599805 12152 model_fn.py:630] Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x00000253E42B7D38>) includes params argument, but params are not passed to Estimator.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

W0529 16:10:37.600802 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

W0529 16:10:37.600802 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
W0529 16:10:37.651666 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
INFO:tensorflow:Reading data files from ./data_dir\translate_up2down-train*
I0529 16:10:37.662636 12152 problem.py:651] Reading data files from ./data_dir\translate_up2down-train*
INFO:tensorflow:partition: 0 num_data_files: 9
I0529 16:10:37.668637 12152 problem.py:677] partition: 0 num_data_files: 9
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
W0529 16:10:37.671612 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

W0529 16:10:37.816255 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

W0529 16:10:38.150365 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
W0529 16:10:39.232438 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:10:39.422930 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
W0529 16:10:39.771997 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

W0529 16:10:40.370483 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:10:40.558941 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
INFO:tensorflow:Calling model_fn.
I0529 16:10:41.336859 12152 estimator.py:1148] Calling model_fn.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0529 16:10:41.337856 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

W0529 16:10:41.337856 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

W0529 16:10:41.337856 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

W0529 16:10:41.350860 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

INFO:tensorflow:Setting T2TModel mode to 'train'
I0529 16:10:41.351856 12152 t2t_model.py:2262] Setting T2TModel mode to 'train'
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0529 16:10:41.352859 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

W0529 16:10:41.479479 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

INFO:tensorflow:Using variable initializer: uniform_unit_scaling
I0529 16:10:42.401330 12152 api.py:332] Using variable initializer: uniform_unit_scaling
INFO:tensorflow:Transforming feature 'inputs' with symbol_modality_3486_256.bottom
I0529 16:10:51.952751 12152 api.py:332] Transforming feature 'inputs' with symbol_modality_3486_256.bottom
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
W0529 16:10:52.518243 12152 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0529 16:10:52.750616 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

INFO:tensorflow:Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
I0529 16:10:54.532984 12152 api.py:332] Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
INFO:tensorflow:Building model body
I0529 16:10:54.662637 12152 api.py:332] Building model body
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

W0529 16:10:57.418312 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0529 16:10:57.628707 12152 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:Entity <function attention_bias_to_padding at 0x00000253DD4E3CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
W0529 16:10:58.450544 12152 ag_logging.py:146] Entity <function attention_bias_to_padding at 0x00000253DD4E3CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

W0529 16:10:59.341177 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

W0529 16:10:59.341177 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

W0529 16:11:03.783540 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

W0529 16:11:03.783540 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:Entity <function layers at 0x00000253D3E77438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x00000253E5EA3548>
W0529 16:11:04.070772 12152 ag_logging.py:146] Entity <function layers at 0x00000253D3E77438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x00000253E5EA3548>
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

W0529 16:11:04.071770 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

W0529 16:11:05.287972 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

INFO:tensorflow:Transforming body output with symbol_modality_3486_256.top
I0529 16:11:12.560762 12152 api.py:332] Transforming body output with symbol_modality_3486_256.top
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

W0529 16:11:13.875245 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

W0529 16:11:17.263191 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

INFO:tensorflow:Base learning rate: 2.000000
I0529 16:11:18.903856 12152 learning_rate.py:29] Base learning rate: 2.000000
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

W0529 16:11:18.911837 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

INFO:tensorflow:Trainable Variables Total size: 4581888
I0529 16:11:18.914894 12152 optimize.py:336] Trainable Variables Total size: 4581888
INFO:tensorflow:Non-trainable variables Total size: 5
I0529 16:11:18.915879 12152 optimize.py:336] Non-trainable variables Total size: 5
INFO:tensorflow:Using optimizer adam
I0529 16:11:18.915879 12152 optimize.py:191] Using optimizer adam
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

W0529 16:11:18.916898 12152 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

INFO:tensorflow:Done calling model_fn.
I0529 16:11:23.022883 12152 estimator.py:1150] Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
I0529 16:11:23.024877 12152 basic_session_run_hooks.py:541] Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
I0529 16:11:36.673000 12152 monitored_session.py:240] Graph was finalized.
2020-05-29 16:11:36.687932: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-05-29 16:11:36.756891: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-05-29 16:11:37.502787: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce MX150 major: 6 minor: 1 memoryClockRate(GHz): 1.0375
pciBusID: 0000:01:00.0
2020-05-29 16:11:37.503508: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-05-29 16:11:37.539237: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:11:37.571183: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-05-29 16:11:37.581827: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-05-29 16:11:37.633096: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-05-29 16:11:37.660589: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-05-29 16:11:37.755912: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-29 16:11:38.032655: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-05-29 16:11:41.463209: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-05-29 16:11:41.464923: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-05-29 16:11:41.465092: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-05-29 16:11:41.484048: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1945 MB memory) -> physical GPU (device: 0, name: GeForce MX150, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from ./output\model.ckpt-0
I0529 16:11:41.554010 12152 saver.py:1284] Restoring parameters from ./output\model.ckpt-0
2020-05-29 16:11:42.482758: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.90G (2040109312 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-05-29 16:11:42.483497: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.71G (1836098304 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
W0529 16:11:42.830596 12152 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
I0529 16:11:43.397316 12152 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0529 16:11:43.603765 12152 session_manager.py:502] Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into ./output\model.ckpt.
I0529 16:12:05.476009 12152 basic_session_run_hooks.py:606] Saving checkpoints for 0 into ./output\model.ckpt.
2020-05-29 16:12:19.368713: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:12:20.674622: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.675878: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.676271: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.677047: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.677441: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.677788: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.683538: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.684120: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.688498: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.688941: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.692517: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.696013: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.696448: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.696812: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.698613: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.700583: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.701511: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.703470: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.703863: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.704217: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.704537: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.706293: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.708943: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.709898: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.711746: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.712109: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.712468: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.714013: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.716038: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.717531: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.719241: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.720049: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.721488: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.724284: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.724651: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.728115: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.728534: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.729156: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.729553: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.730783: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.731172: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.731736: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.732124: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.732747: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.733136: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.733587: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.734005: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.735186: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.735590: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.736205: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.736580: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.737346: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.737838: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.738449: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.738891: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.740264: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.740680: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.741220: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.741621: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.744036: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.744608: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.745464: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.745969: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.746691: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.747080: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.747566: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.747959: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.749792: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.750186: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.750582: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.751174: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.751997: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.752564: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.753274: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.753762: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.755432: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.755868: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.756408: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.756846: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.758498: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.758942: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.759981: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.760607: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.762010: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.762436: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.763119: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.763599: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.764928: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.765305: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.765835: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.766217: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.767580: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.767954: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.768484: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.768852: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.769433: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.769824: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.770285: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:20.770823: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:21.234901: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:21.235239: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:12:21.235369: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
2020-05-29 16:12:21.235521: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
Traceback (most recent call last):
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1365, in _do_call
    return fn(*args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(28, 256), b.shape=(256, 256), m=28, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(28, 256), b.shape=(256, 256), m=28, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1195, in _train_model_default
    saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1494, in _train_with_estimator_spec
    _, loss = mon_sess.run([estimator_spec.train_op, estimator_spec.loss])
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 754, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1259, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1360, in run
    raise six.reraise(*original_exc_info)
  File "D:\Anaconda\envs\NLP\lib\site-packages\six.py", line 703, in reraise
    raise value
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1345, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1418, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1176, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 956, in run
    run_metadata_ptr)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1180, in _run
    feed_dict_tensor, options, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1359, in _do_run
    run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1384, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(28, 256), b.shape=(256, 256), m=28, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(28, 256), b.shape=(256, 256), m=28, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
0 successful operations.
0 derived errors ignored.

Original stack trace for 'transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul':
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1191, in _train_model_default
    features, labels, ModeKeys.TRAIN, self.config)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1149, in _call_model_fn
    model_fn_results = self._model_fn(features=features, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1420, in wrapping_model_fn
    use_tpu=use_tpu)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1485, in estimator_model_fn
    logits, losses_dict = model(features)  # pylint: disable=not-callable
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 325, in call
    sharded_logits, losses = self.model_fn_sharded(sharded_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 365, in model_fn_sharded
    if self.use_body_sharded():
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 402, in model_fn_sharded
    sharded_logits, sharded_losses = dp(self.model_fn, datashard_to_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 171, in __call__
    for i in range(self.n):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 229, in __call__
    if self._devices[i] != DEFAULT_DEV_STRING:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 231, in __call__
    outputs.append(fns[i](*my_args[i], **my_kwargs[i]))
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 429, in model_fn
    body_out = self.body(transformed_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 243, in body
    if self.has_input:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 246, in body
    encoder_output, encoder_decoder_attention_bias = self.encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 200, in encode
    return transformer_encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 103, in transformer_encode
    encoder_output = encoder_function(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 201, in transformer_encoder
    for layer in range(hparams.num_encoder_layers or hparams.num_hidden_layers):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 212, in transformer_encoder
    y = common_attention.multihead_attention(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4649, in multihead_attention
    if cache is None or memory_antecedent is None:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4650, in multihead_attention
    q, k, v = compute_qkv(query_antecedent, memory_antecedent,
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4454, in compute_qkv
    q = compute_attention_component(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4399, in compute_attention_component
    if vars_3d_num_heads > 0:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4414, in compute_attention_component
    if filter_width == 1:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4415, in compute_attention_component
    return common_layers.dense(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py", line 3083, in dense
    activations = layers().Dense(units, **kwargs)(x)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\layers\core.py", line 1039, in call
    outputs = standard_ops.tensordot(inputs, self.kernel, [[rank - 1], [0]])
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 4071, in tensordot
    ab_matmul = matmul(a_reshape, b_reshape)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\dispatch.py", line 180, in wrapper
    return target(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 2754, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\gen_math_ops.py", line 6136, in mat_mul
    name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()

2020-05-29 16:12:44.594928: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:187: The name tf.train.Optimizer is deprecated. Please use tf.compat.v1.train.Optimizer instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\research\neural_stack.py:52: The name tf.nn.rnn_cell.RNNCell is deprecated. Please use tf.compat.v1.nn.rnn_cell.RNNCell instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:111: The name tf.OptimizerOptions is deprecated. Please use tf.compat.v1.OptimizerOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_gan\python\estimator\tpu_gan_estimator.py:42: The name tf.estimator.tpu.TPUEstimator is deprecated. Please use tf.compat.v1.estimator.tpu.TPUEstimator instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:32: The name tf.logging.INFO is deprecated. Please use tf.compat.v1.logging.INFO instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

WARNING:tensorflow:From D:/Anaconda/envs/NLP/Scripts/t2t-trainer:33: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

INFO:tensorflow:Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
I0529 16:12:53.545016  6292 usr_dir.py:43] Importing user module usr_dir from path C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP
INFO:tensorflow:Loading hparams from existing json ./output\hparams.json
I0529 16:12:53.547978  6292 hparams_lib.py:64] Loading hparams from existing json ./output\hparams.json
INFO:tensorflow:Overwrite key batch_size: 4096 -> 32
I0529 16:12:53.559947  6292 hparams_lib.py:86] Overwrite key batch_size: 4096 -> 32
INFO:tensorflow:Overriding hparams in transformer_small with batch_size=1
I0529 16:12:53.560945  6292 hparams_lib.py:55] Overriding hparams in transformer_small with batch_size=1
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

W0529 16:12:53.560945  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:844: The name tf.set_random_seed is deprecated. Please use tf.compat.v1.set_random_seed instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

W0529 16:12:53.562939  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:123: The name tf.GraphOptions is deprecated. Please use tf.compat.v1.GraphOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

W0529 16:12:53.562939  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:129: The name tf.GPUOptions is deprecated. Please use tf.compat.v1.GPUOptions instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0529 16:12:53.562939  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:131: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
W0529 16:12:53.562939  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:241: RunConfig.__init__ (from tensorflow.contrib.learn.python.learn.estimators.run_config) is deprecated and will be removed in a future version.
Instructions for updating:
When switching to tf.estimator.Estimator, use tf.estimator.RunConfig instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

W0529 16:12:53.563936  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py:264: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.

INFO:tensorflow:Configuring DataParallelism to replicate the model.
I0529 16:12:53.563936  6292 trainer_lib.py:264] Configuring DataParallelism to replicate the model.
INFO:tensorflow:schedule=train
I0529 16:12:53.563936  6292 devices.py:76] schedule=train
INFO:tensorflow:worker_gpu=1
I0529 16:12:53.563936  6292 devices.py:77] worker_gpu=1
INFO:tensorflow:sync=False
I0529 16:12:53.563936  6292 devices.py:78] sync=False
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

W0529 16:12:53.563936  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\devices.py:139: The name tf.logging.warn is deprecated. Please use tf.compat.v1.logging.warn instead.

WARNING:tensorflow:Schedule=train. Assuming that training is running on a single machine.
W0529 16:12:53.563936  6292 devices.py:141] Schedule=train. Assuming that training is running on a single machine.
INFO:tensorflow:datashard_devices: ['gpu:0']
I0529 16:12:53.563936  6292 devices.py:170] datashard_devices: ['gpu:0']
INFO:tensorflow:caching_devices: None
I0529 16:12:53.563936  6292 devices.py:171] caching_devices: None
INFO:tensorflow:ps_devices: ['gpu:0']
I0529 16:12:53.563936  6292 devices.py:172] ps_devices: ['gpu:0']
I0529 16:12:53.564933  6292 merge_vocab.py:101] Loading and processing vocabulary from: C:\Users\zhang\OneDrive\Desktop\NLP\MyNLP\usr_dir/merge.txt.vocab.clean
I0529 16:12:53.576902  6292 merge_vocab.py:105] Done
INFO:tensorflow:Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x000002026CBB1D08>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x000002026CBAEC48>}
I0529 16:12:53.578896  6292 estimator.py:212] Using config: {'_task_type': None, '_task_id': 0, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x000002026CBB1D08>, '_master': '', '_num_ps_replicas': 0, '_num_worker_replicas': 0, '_environment': 'local', '_is_chief': True, '_evaluation_master': '', '_train_distribute': None, '_eval_distribute': None, '_experimental_max_worker_delay_secs': None, '_device_fn': None, '_tf_config': gpu_options {
  per_process_gpu_memory_fraction: 1.0
}
, '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_secs': 1800, '_log_step_count_steps': 100, '_protocol': None, '_session_config': gpu_options {
  per_process_gpu_memory_fraction: 0.95
}
allow_soft_placement: true
graph_options {
  optimizer_options {
    global_jit_level: OFF
  }
}
isolate_session_state: true
, '_save_checkpoints_steps': None, '_keep_checkpoint_max': 1000, '_keep_checkpoint_every_n_hours': 10000, '_model_dir': './output', '_session_creation_timeout_secs': 7200, 'use_tpu': False, 't2t_device_info': {'num_async_replicas': 1}, 'data_parallelism': <tensor2tensor.utils.expert_utils.Parallelism object at 0x000002026CBAEC48>}
WARNING:tensorflow:Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x000002026CB97D38>) includes params argument, but params are not passed to Estimator.
W0529 16:12:53.578896  6292 model_fn.py:630] Estimator's model_fn (<function T2TModel.make_estimator_model_fn.<locals>.wrapping_model_fn at 0x000002026CB97D38>) includes params argument, but params are not passed to Estimator.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

W0529 16:12:53.579894  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:331: The name tf.gfile.Exists is deprecated. Please use tf.io.gfile.exists instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

W0529 16:12:53.579894  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py:348: The name tf.gfile.Open is deprecated. Please use tf.io.gfile.GFile instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
W0529 16:12:53.604827  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\training_util.py:236: Variable.initialized_value (from tensorflow.python.ops.variables) is deprecated and will be removed in a future version.
Instructions for updating:
Use Variable.read_value. Variables in 2.X are initialized automatically both in eager and graph (inside tf.defun) contexts.
INFO:tensorflow:Reading data files from ./data_dir\translate_up2down-train*
I0529 16:12:53.612807  6292 problem.py:651] Reading data files from ./data_dir\translate_up2down-train*
INFO:tensorflow:partition: 0 num_data_files: 9
I0529 16:12:53.615796  6292 problem.py:677] partition: 0 num_data_files: 9
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
W0529 16:12:53.618820  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:687: parallel_interleave (from tensorflow.python.data.experimental.ops.interleave_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.data.Dataset.interleave(map_func, cycle_length, block_length, num_parallel_calls=tf.data.experimental.AUTOTUNE)` instead. If sloppy execution is desired, use `tf.data.Options.experimental_determinstic`.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

W0529 16:12:53.761408  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\data_generators\problem.py:712: The name tf.FixedLenFeature is deprecated. Please use tf.io.FixedLenFeature instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

W0529 16:12:54.110514  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.logging.warning is deprecated. Please use tf.compat.v1.logging.warning instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
W0529 16:12:55.138726  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:276: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and: 
`tf.data.TFRecordDataset(path)`
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:12:55.364122  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:38: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
W0529 16:12:55.677324  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\data\experimental\ops\grouping.py:194: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

W0529 16:12:56.375451  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
W0529 16:12:56.561960  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\data_reader.py:234: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
INFO:tensorflow:Calling model_fn.
I0529 16:12:56.906050  6292 estimator.py:1148] Calling model_fn.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0529 16:12:56.906050  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1991: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

W0529 16:12:56.906999  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1994: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

W0529 16:12:56.906999  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1997: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

W0529 16:12:56.920999  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:1457: The name tf.get_variable_scope is deprecated. Please use tf.compat.v1.get_variable_scope instead.

INFO:tensorflow:Setting T2TModel mode to 'train'
I0529 16:12:56.920999  6292 t2t_model.py:2262] Setting T2TModel mode to 'train'
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0529 16:12:56.921959  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:100: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

W0529 16:12:57.047677  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py:245: The name tf.summary.text is deprecated. Please use tf.compat.v1.summary.text instead.

INFO:tensorflow:Using variable initializer: uniform_unit_scaling
I0529 16:12:58.075905  6292 api.py:332] Using variable initializer: uniform_unit_scaling
INFO:tensorflow:Transforming feature 'inputs' with symbol_modality_3486_256.bottom
I0529 16:13:07.917770  6292 api.py:332] Transforming feature 'inputs' with symbol_modality_3486_256.bottom
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
W0529 16:13:08.521153  6292 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0529 16:13:08.758517  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

INFO:tensorflow:Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
I0529 16:13:10.610804  6292 api.py:332] Transforming feature 'targets' with symbol_modality_3486_256.targets_bottom
INFO:tensorflow:Building model body
I0529 16:13:10.741450  6292 api.py:332] Building model body
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

W0529 16:13:13.455804  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.mod is deprecated. Please use tf.math.mod instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
W0529 16:13:13.669295  6292 deprecation.py:506] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py:95: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:Entity <function attention_bias_to_padding at 0x0000020265DC3CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
W0529 16:13:14.524071  6292 ag_logging.py:146] Entity <function attention_bias_to_padding at 0x0000020265DC3CA8> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: 
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

W0529 16:13:15.444592  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.squared_difference is deprecated. Please use tf.math.squared_difference instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

W0529 16:13:15.444592  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.rsqrt is deprecated. Please use tf.math.rsqrt instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

W0529 16:13:20.302641  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

W0529 16:13:20.303645  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

WARNING:tensorflow:Entity <function layers at 0x000002025C757438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x000002026E5E5908>
W0529 16:13:20.592884  6292 ag_logging.py:146] Entity <function layers at 0x000002025C757438> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: If not matching a CFG node, must be a block statement: <gast.gast.ImportFrom object at 0x000002026E5E5908>
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

W0529 16:13:20.593881  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py:3083: The name tf.layers.Dense is deprecated. Please use tf.compat.v1.layers.Dense instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

W0529 16:13:21.650043  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.image is deprecated. Please use tf.compat.v1.summary.image instead.

INFO:tensorflow:Transforming body output with symbol_modality_3486_256.top
I0529 16:13:29.396347  6292 api.py:332] Transforming body output with symbol_modality_3486_256.top
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

W0529 16:13:30.729768  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.

WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

W0529 16:13:34.551550  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\converters\directives.py:119: The name tf.log is deprecated. Please use tf.math.log instead.

INFO:tensorflow:Base learning rate: 2.000000
I0529 16:13:36.323815  6292 learning_rate.py:29] Base learning rate: 2.000000
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

W0529 16:13:36.330797  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:250: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.

INFO:tensorflow:Trainable Variables Total size: 4581888
I0529 16:13:36.333749  6292 optimize.py:336] Trainable Variables Total size: 4581888
INFO:tensorflow:Non-trainable variables Total size: 5
I0529 16:13:36.333749  6292 optimize.py:336] Non-trainable variables Total size: 5
INFO:tensorflow:Using optimizer adam
I0529 16:13:36.334746  6292 optimize.py:191] Using optimizer adam
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

W0529 16:13:36.335744  6292 module_wrapper.py:139] From D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\optimize.py:102: The name tf.train.get_or_create_global_step is deprecated. Please use tf.compat.v1.train.get_or_create_global_step instead.

INFO:tensorflow:Done calling model_fn.
I0529 16:13:40.558998  6292 estimator.py:1150] Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
I0529 16:13:40.559993  6292 basic_session_run_hooks.py:541] Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
I0529 16:13:52.303823  6292 monitored_session.py:240] Graph was finalized.
2020-05-29 16:13:52.309984: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-05-29 16:13:52.394312: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-05-29 16:13:53.129148: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: GeForce MX150 major: 6 minor: 1 memoryClockRate(GHz): 1.0375
pciBusID: 0000:01:00.0
2020-05-29 16:13:53.129523: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-05-29 16:13:53.193676: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:13:53.245202: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_100.dll
2020-05-29 16:13:53.278358: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_100.dll
2020-05-29 16:13:53.350059: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_100.dll
2020-05-29 16:13:53.398878: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_100.dll
2020-05-29 16:13:53.512896: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-29 16:13:53.727502: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-05-29 16:13:56.747562: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-05-29 16:13:56.747918: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-05-29 16:13:56.748084: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 
2020-05-29 16:13:56.753598: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1945 MB memory) -> physical GPU (device: 0, name: GeForce MX150, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from ./output\model.ckpt-0
I0529 16:13:56.767719  6292 saver.py:1284] Restoring parameters from ./output\model.ckpt-0
2020-05-29 16:13:57.513730: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.90G (2040109312 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
2020-05-29 16:13:57.514654: I tensorflow/stream_executor/cuda/cuda_driver.cc:831] failed to allocate 1.71G (1836098304 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY: out of memory
WARNING:tensorflow:From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
W0529 16:13:57.874982  6292 deprecation.py:323] From D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\saver.py:1069: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
I0529 16:13:58.426804  6292 session_manager.py:500] Running local_init_op.
INFO:tensorflow:Done running local_init_op.
I0529 16:13:58.636232  6292 session_manager.py:502] Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into ./output\model.ckpt.
I0529 16:14:19.506845  6292 basic_session_run_hooks.py:606] Saving checkpoints for 0 into ./output\model.ckpt.
2020-05-29 16:14:32.268979: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-05-29 16:14:32.995729: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:32.996755: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:32.997053: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:32.997599: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:32.997979: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:32.998321: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.003611: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.004160: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.008061: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.008454: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.012723: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.015881: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.016285: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.016706: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.018174: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.019852: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.021042: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.022560: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.022959: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.023339: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.023768: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.025549: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.027141: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.027914: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.029960: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.030468: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.030808: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.032308: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.033932: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.035082: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.036408: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.037107: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.038378: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.041159: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.041544: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.045416: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.046905: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.047606: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.048104: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.049331: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.049797: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.050329: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.050715: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.051223: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.051602: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.051973: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.052350: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.053432: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.053763: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.054304: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.054635: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.055166: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.055555: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.056033: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.056416: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.057786: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.058189: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.058665: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.059128: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.060605: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.061037: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.061600: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.062419: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.063292: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.063606: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.064120: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.064487: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.066278: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.066655: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.067078: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.067397: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.067958: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.068358: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.068879: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.069252: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.071064: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.071482: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.072025: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.072396: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.073836: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.074157: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.074739: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.075050: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.075879: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.076198: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.076718: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.077028: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.078272: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.078645: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.080445: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.081013: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.082260: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.082650: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.083107: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.083422: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.083930: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.084243: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.084606: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.084920: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.422380: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.422736: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_ALLOC_FAILED
2020-05-29 16:14:33.422869: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
2020-05-29 16:14:33.423037: W tensorflow/stream_executor/stream.cc:2041] attempting to perform BLAS operation using StreamExecutor without BLAS support
Traceback (most recent call last):
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1365, in _do_call
    return fn(*args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(6, 256), b.shape=(256, 256), m=6, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(6, 256), b.shape=(256, 256), m=6, n=256, k=256
	 [[{{node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul}}]]
0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "D:\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "D:/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1195, in _train_model_default
    saving_listeners)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1494, in _train_with_estimator_spec
    _, loss = mon_sess.run([estimator_spec.train_op, estimator_spec.loss])
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 754, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1259, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1360, in run
    raise six.reraise(*original_exc_info)
  File "D:\Anaconda\envs\NLP\lib\site-packages\six.py", line 703, in reraise
    raise value
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1345, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1418, in run
    run_metadata=run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\training\monitored_session.py", line 1176, in run
    return self._sess.run(*args, **kwargs)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 956, in run
    run_metadata_ptr)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1180, in _run
    feed_dict_tensor, options, run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1359, in _do_run
    run_metadata)
  File "D:\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\client\session.py", line 1384, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InternalError: 2 root error(s) found.
  (0) Internal: Blas GEMM launch failed : a.shape=(6, 256), b.shape=(256, 256), m=6, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
	 [[transformer/parallel_0_5/transformer/transformer/body/decoder/layer_0/encdec_attention/multihead_attention/dot_product_attention/Max/_2299]]
  (1) Internal: Blas GEMM launch failed : a.shape=(6, 256), b.shape=(256, 256), m=6, n=256, k=256
	 [[node transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul (defined at \Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py:1748) ]]
0 successful operations.
0 derived errors ignored.

Original stack trace for 'transformer/parallel_0_5/transformer/transformer/body/encoder/layer_0/self_attention/multihead_attention/q/Tensordot/MatMul':
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 33, in <module>
    tf.app.run()
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\platform\app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 299, in run
    _run_main(main, args)
  File "\Anaconda\envs\NLP\lib\site-packages\absl\app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/Anaconda/envs/NLP/Scripts/t2t-trainer", line 28, in main
    t2t_trainer.main(argv)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 418, in main
    execute_schedule(exp)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\bin\t2t_trainer.py", line 371, in execute_schedule
    getattr(exp, FLAGS.schedule)()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\trainer_lib.py", line 477, in train
    max_steps=max_steps or self._train_spec.max_steps)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 370, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1161, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1191, in _train_model_default
    features, labels, ModeKeys.TRAIN, self.config)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_estimator\python\estimator\estimator.py", line 1149, in _call_model_fn
    model_fn_results = self._model_fn(features=features, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1420, in wrapping_model_fn
    use_tpu=use_tpu)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 1485, in estimator_model_fn
    logits, losses_dict = model(features)  # pylint: disable=not-callable
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 325, in call
    sharded_logits, losses = self.model_fn_sharded(sharded_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 365, in model_fn_sharded
    if self.use_body_sharded():
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 402, in model_fn_sharded
    sharded_logits, sharded_losses = dp(self.model_fn, datashard_to_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 171, in __call__
    for i in range(self.n):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 229, in __call__
    if self._devices[i] != DEFAULT_DEV_STRING:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\expert_utils.py", line 231, in __call__
    outputs.append(fns[i](*my_args[i], **my_kwargs[i]))
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\utils\t2t_model.py", line 429, in model_fn
    body_out = self.body(transformed_features)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 243, in body
    if self.has_input:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 246, in body
    encoder_output, encoder_decoder_attention_bias = self.encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 200, in encode
    return transformer_encode(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\models\transformer.py", line 103, in transformer_encode
    encoder_output = encoder_function(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 201, in transformer_encoder
    for layer in range(hparams.num_encoder_layers or hparams.num_hidden_layers):
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 339, in for_stmt
    return _py_for_stmt(iter_, extra_test, body, get_state, set_state, init_vars)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 350, in _py_for_stmt
    state = body(target, *state)
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\transformer_layers.py", line 212, in transformer_encoder
    y = common_attention.multihead_attention(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4649, in multihead_attention
    if cache is None or memory_antecedent is None:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4650, in multihead_attention
    q, k, v = compute_qkv(query_antecedent, memory_antecedent,
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4454, in compute_qkv
    q = compute_attention_component(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4399, in compute_attention_component
    if vars_3d_num_heads > 0:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4414, in compute_attention_component
    if filter_width == 1:
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 895, in if_stmt
    return _py_if_stmt(cond, body, orelse)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\autograph\operators\control_flow.py", line 1004, in _py_if_stmt
    return body() if cond else orelse()
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_attention.py", line 4415, in compute_attention_component
    return common_layers.dense(
  File "\Anaconda\envs\NLP\lib\site-packages\tensor2tensor\layers\common_layers.py", line 3083, in dense
    activations = layers().Dense(units, **kwargs)(x)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\layers\base.py", line 548, in __call__
    outputs = super(Layer, self).__call__(inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\engine\base_layer.py", line 854, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\keras\layers\core.py", line 1039, in call
    outputs = standard_ops.tensordot(inputs, self.kernel, [[rank - 1], [0]])
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 4071, in tensordot
    ab_matmul = matmul(a_reshape, b_reshape)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\dispatch.py", line 180, in wrapper
    return target(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\math_ops.py", line 2754, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\ops\gen_math_ops.py", line 6136, in mat_mul
    name=name)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\op_def_library.py", line 794, in _apply_op_helper
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3357, in create_op
    attrs, op_def, compute_device)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 3426, in _create_op_internal
    op_def=op_def)
  File "\Anaconda\envs\NLP\lib\site-packages\tensorflow_core\python\framework\ops.py", line 1748, in __init__
    self._traceback = tf_stack.extract_stack()

